### ISSUE TITLE
Allowing the user to run "cleanup" manually

### ISSUE THREAD

**Issue by dahankzter** [scylladb/scylla-operator#49]
We can support this action by marking a node for cleanup and invoking the REST API.
We need to track the node state to avoid invoking the command several times but other than that there shouldn't be much to do.

**Comment by dahankzter** [comment]
A simple fallback would be to describe to the user how to run it on a particular node by simple docker exec nodetool command. If this turns out to be messy at least.

**Comment by dahankzter** [comment]
Perhaps we can automate this after all. What if we after a `scale up` simply launch a `cleanup` action once the cluster reaches `UN`. This could either be done in parallel which would be simpler or one at a time which require a little bit more coordination. What do you think @tzach @slivne? The work isn't that big and it's a nice touch to automatically save the users disk. space

**Comment by dahankzter** [comment]
Hmmm perhaps it isn't such a good idea to cleanup automatically. What if a user want's to scale up as a consequence of high load. Then they might not want to bog down the system with compactions.
One at a time might be ok and possibly preferable but in parallel I think is unacceptable.

**Comment by gnumoreno** [comment]
@dahankzter here is my opinion based on experience from the field:

Clean up should be part of scylla-manager. Also, since sometimes people use it because the cluster had too much data (over 50% disk usage in the nodes and using STCS) running a node-wide cleanup can lead to disk full.

Ideally for reasons of disk usage AND parallelism (as you mentioned above) cleanup should be run by keyspace/table and from smallest to biggest. 

For each keyspace (again smallest to biggest, one node at a time), something like this
```
cd /var/lib/scylla/data/$KEYSPACE
for i in $(du -d2 -h | sort -h | cut -d'/' -f2 | cut -d'-' -f1 | grep -Ev '^[0-9]'); do echo nodetool cleanup -- $(pwd | cut -d"/" -f6) $i; done
```
Of course this is just an example on how to do it manually. I'm not suggesting we use shellscript.

**Comment by dahankzter** [comment]
Great input @gnumoreno! It is probably better to defer implementing this now and simply document a few examples of how to do this in a k8s setting. Perhaps less elaborate than your example but at least something to help people save some space.

**Comment by dahankzter** [comment]
What do you think @tzach? Postpone implementing this for now and just device and document a workable way to run this by hand as needed?

**Comment by dahankzter** [comment]
Let's restrict this one to a docs issue for now. We can revisit when manager has a `cleanup` command.
